src/components/custom/signup.tsx

```diff
@@ -69,8 +68,16 @@ export const Appearances = {
     public: 'public',
     newSignup: 'newSignup',
 }
+const query = graphql`
+    query {
+        view_email: file(relativePath: { eq: "sign-up/response-email.png" }) {
+            ...fadeIn
```

Comment:
Can you try to follow this approach? We have many issues due the `fadeIn `
https://github.com/binary-com/deriv-com/pull/4308/files#diff-e9c589d0bf916f355751f5ae93ba4e88eeb7e6542683ded00bd7141de9dc6be6